support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -45,15 +47,36 @@ const shorterBoxMargin = css`
 	}
 `;
 
+function setAbandonedBasketCookie(
+	product: string, 
+	amount: number, 
+	billingPeriod: string, 
+	region: string
+){
+	const abandonedBasket = {
+		product,
+		amount,
+		billingPeriod,
+		region
+	};
+
+	const COOKIE_EXPIRY_DAYS = 3;
+	useEffect(() => {
+		cookie.set('abandonedBasket', JSON.stringify(abandonedBasket), COOKIE_EXPIRY_DAYS)
```

Comment:
Just checking that we've cleared this with the consent team?

Reply:
It's classified as an essential cookie and it's been cleared with legal and DPO, and it's in the process of being added to CMP

Reply:
It's now added to the CMP as well